packages/astro/src/core/config/schema.ts

```diff
@@ -133,7 +133,7 @@ export const AstroConfigSchema = z.object({
 		.optional()
 		.default(ASTRO_CONFIG_DEFAULTS.trailingSlash),
 	output: z
-		.union([z.literal('static'), z.literal('server'), z.literal('hybrid')])
+		.union([z.literal('static'), z.literal('server')])
```

Comment:
The names here honestly don't matter, since they're not used for logic anymore apart from choosing the default value of prerender

My own take is that the key should be changed to something else than output, Matthew felt like it was too confusing for users if hybrid was the default etc. It's still up for debate!

Reply:
Yeah, I'd prefer to have a boolean `prerenderDefault`, which defaults to `false`. No more `output`.

Reply:
`prerenderDefault` forces the concept of prerendering onto people who just want to build a regular static (or server) site. People who build mixed sites are the exception.

I don't feel super strongly about `hybrid` vs `static` personally. I think it's ok that `static` has some non-static output because `server` can already have non-server output too.